cfgov/unprocessed/js/routes/on-demand/summary-mobile.js

```diff
@@ -0,0 +1,8 @@
+import SummaryMobile from '../../organisms/SummaryMobile.js';
+
+const summaryMobileDoms = document.querySelectorAll( `.${ SummaryMobile.BASE_CLASS }` );
+let summaryMobile;
+for ( let i = 0, len = summaryMobileDoms.length; i < len; i++ ) {
+  summaryMobile = new SummaryMobile( summaryMobileDoms[i] );
+  summaryMobile.init();
+}
```

Comment:
Could we use [`instantiateAll`](https://github.com/cfpb/consumerfinance.gov/blob/0b7b435eb1894e8a985cf867409c3ba0693752aa/cfgov/unprocessed/js/routes/on-demand/video-player.js#L6-L8) here?

Reply:
Hmm unfortunately I see a bug which means we can't. `instantiateAll` [looks for `data-js-hook`](https://github.com/cfpb/design-system/blob/main/packages/cfpb-atomic-component/src/utilities/atomic-helpers.js#L130). This is meant to avoid double initialization of a component. Instead it should be checking for the existance `state_atomic_init`, not just `data-js-hook`.

Reply:
https://github.com/cfpb/design-system/issues/1157

Reply:
Updated in https://github.com/cfpb/consumerfinance.gov/pull/6206/commits/595e990bc7bd9d9cd037f9e2f8f3c5e841d287ea